py/server/deephaven/table.py

```diff
@@ -485,6 +688,13 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def _encode_signature(fn: Callable) -> str:
+    """Encode the signature of a function into a string for backwards compatibility with the vectorization implementation
+    in the server."""
+    p_sig = _parse_signature(fn)
+    return re.sub("[\[N,]", "", p_sig.encoded)
```

Comment:
Backwards compatibility seems weird for an internal feature.  Should things be done differently?